src/Types/RealFloatType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+final class RealFloatType extends Type
```

Comment:
```suggestion
final class SmallFloatType extends Type
```

"real" is quite hard to understand as "real" in computer science means very often float in general. I propose to name it like "float single precison", "float 32", "smallfloat". The last would be quite consistent with "smallint".

Reply:
Yeah, I thought about that too when I found out that MySQL handles it differently :). But which name is better to choose? I was thinking of **SingleFloat**, but maybe your suggestion **SmallFloat** is more intuitive (I think float32 might not be very clear for many people).

Reply:
Renamed to **SmallFloat**. Thanks!